### ISSUE TITLE
Increase expand row icon target size for accessibility standards on `DataTable`

### ISSUE THREAD

**Issue by spli02** [fortanix/baklava#642]
Currently, the interactive target for the `DataTable's` expand row icon is only 15 × 20.8 pixels. 
This does not meet the accessibility requirement of at least 24 × 24 CSS pixels.
We should increase the target size to ensure that users can toggle row expansion more easily.

<img width="578" height="289" alt="Image" src="https://github.com/user-attachments/assets/03f2496d-b514-461e-8b46-fb72d45db368" />

Ref: https://github.com/fortanix/baklava/issues/485

**Comment by mkrause** [comment]
I believe this is fixed in https://github.com/fortanix/baklava/pull/648
